File: demo/src/main/java/exercise1/CustomerService.java
package exercise1;

import java.util.List;

public interface CustomerService {
	public CustomerBoundary create(CustomerBoundary consumer);

	public CustomerBoundary getByEmail(String email);

	public CustomerBoundary login(String email, String password);

	public void update(String email, CustomerBoundary consumer);

	public void deleteAll();

	public List<CustomerBoundary> getAllCustomers(int size, int page, String sortAttribute, String order,String criteriaType,String criteriaValue);
}

File: New folder/Demo - Kotlin/src/main/kotlin/demo/CustomerService.kt
package demo


interface CustomerService {
    fun create(consumer: CustomerBoundary?): CustomerBoundary?
    fun getByEmail(email: String?): CustomerBoundary?
    fun login(email: String?, password: String?): CustomerBoundary?
    fun update(email: String?, consumer: CustomerBoundary?)
    fun deleteAll()
    fun getAllCustomers(
        size: Int, page: Int, sortAttribute: String?, order: String?,
        criteriaType: String?, criteriaValue: String?
    ): List<CustomerBoundary?>?

}